Transport/TransportParams.H

```diff
@@ -105,7 +117,11 @@ struct InitTransParm<EOSType, SimpleTransport>
     egtransetCOFETA(tparm->fitmu.data());
     egtransetCOFLAM(tparm->fitlam.data());
     egtransetCOFD(tparm->fitdbin.data());
-
+    amrex::ParmParse pp("transport");
+    pp.query("use_soret", tparm->use_soret);
+    egtransetNLITE(&tparm->numLite);
```

Comment:
I'll put a flag in to catch this potential memory issue - as for the default number the only common species that will ever be classed as light will be H, H2 and He. Only exceptions I can think of are isotopes of hydrogen, and I'm also not sure how the mechanism deals with the case of electron transport whens compiled with electric field capabilities. Either way, I can almost certainly reduce this number, and it would be nice if the NLITE was #define'd in mechanism, so this could be hardcoded at compile time rather than the way it is currently done.